docs/capabilities/app-client.md

```diff
@@ -2,17 +2,52 @@
 
 Application client that works with ARC-0032 application spec defined smart contracts (e.g. via Beaker).
 
+> [!NOTE]
+> This page covers the untyped app client, but it's worth exploring the [typed client generation feature in AlgoKit](https://github.com/algorandfoundation/algokit-cli/blob/main/docs/features/generate.md#1-typed-clients), which will give you a better developer experience with strong typing and intellisense specific to the app itself.
+
```

Comment:
This is actually a huge gap in the AlgoKit Utils docs. I think it's important in a future pull requests that focusses on apps that we pull in typed app client docs - they are currently spread between the generator repo and the algokit cli repo.